cranelift/README.md

```diff
@@ -27,6 +27,11 @@ For an example of how to use Cranelift to run WebAssembly code, see
 
 [Wasmtime]: https://github.com/bytecodealliance/wasmtime
 
+For various examples of how to use Cranelift to accomplish common compiler
+tasks, see the [Cranelift Examples] repository.
```

Comment:
Let's make this a little more explicit about the third-party maintenance -- perhaps something like "For various examples of how to use Cranelift to accomplish common compiler tasks, in addition to the above, @simvux maintains a [Cranelift Examples] repository." ?

Reply:
Good point! 

Although not sure about the wording. "in addition to above" could also imply that I maintain the JIT demo.

Reply:
Sounds good!